src/debugAdapter/types/web3.d.ts

```diff
@@ -40,6 +40,8 @@ declare namespace Web3 {
     getBlockNumber: any;
     currentProvider: any;
     getAccounts: any;
+    getBalance: any;
```

Comment:
Not related to this PR in particular, but do we need to add explicit typings for `web3js`? `web3js` already comes with typings, can we use the one provided instead?

For instance, the type `Web3.IBatchRequest` in `web3Wrapper.ts` could be replaced by `InstanceType<Web3['BatchRequest']>`.

Reply:
@acuarica yes, could be better explicit the types. I'm gonna change those types

Thanks

Reply:
Hey @acuarica,

You're right. We need to refactor it. Sincerely, I don't know why we have the web3.d.ts. We need to open one issue for that

Thank you